music_assistant/config.yaml

```diff
@@ -13,7 +13,7 @@ host_network: true
 ingress: true
 ingress_port: 8095
 panel_icon: mdi:bookshelf
-panel_title: Music Assistant
+panel_title: Music Assistant Server
```

Comment:
we also have the beta and dev addons - add the server postfix there as well ?

Reply:
Ok now I am with you. Those are also updated.